src/qibolab/_core/execution_parameters.py

```diff
@@ -1,8 +1,10 @@
 from enum import Enum, auto
 from typing import Any, Optional
 
-from qibolab.serialize import Model
-from qibolab.sweeper import ParallelSweepers
+from .serialize import Model
+from .sweeper import ParallelSweepers
+
+__all__ = ["AcquisitionType", "AveragingMode", "ExecutionParameters"]
```

Comment:
Unfortunately, it is either here or in 0.3.

I was considering doing it here, but then I thought "too late, better not to rush". I wonder if it's sufficiently advantageous that I should still consider doing it here...

Reply:
I would expect the change to be pretty minimal: use `kwargs` instead of `options` in `Platform.execute(sequences, sweepers=None, **kwargs)` and do
```py
options = ExecutionParameters(**kwargs)
```
right after. Also, remove `ExecutionParameters` from the public API and I guess some updates in the docs/README. If it's only that we could probably do in the short time we have, if it ends up being more complicated certainly no.

Whether it's sufficiently advantageous I am not sure. It should make `Platform.execute` slightly easier to use, but even this may be subjective.

Reply:
> If it's only that we could probably do in the short time we have, if it ends up being more complicated certainly no.

Then I'll try

> Btw, I could also do that, if you agree.

Your help will be precious in double-checking I haven't broken anything :P

Reply:
Postponed "by 1 PR", #1047